Test/Framework/Graphics/SpriteBatchTest.cs

```diff
@@ -296,7 +294,7 @@ public void DrawWithTexture()
         [Test]
         public void DrawWithCustomEffectAndTwoTextures()
         {
-            var customSpriteEffect = AssetTestUtility.CompileEffect(gd, "CustomSpriteBatchEffect.fx");
+            var customSpriteEffect = content.Load<Effect>(Paths.Effect("CustomSpriteBatchEffect"));
```

Comment:
> How about we change these to AssetTestUtility.LoadEffecr and in there do

Yeah I think having the `AssetTestUtility.LoadEffect` which internally builds the effect on desktop platforms that support effect content building or loads the effect on platforms that do not...  is the best path forward.